workflows/cohort/tasks/slivar.wdl

```diff
@@ -562,7 +526,7 @@ workflow slivar {
       comphet_vcf = slivar_compound_hets_bgzip_vcf.vcf_gz_output,
       ped = ped,
       lof_lookup = generate_lof_lookup.lof_lookup,
```

Comment:
Just to be sure, why are we directly providing the clinvar lookup, but rebuilding the ensembl gff and lof lookup every time?

Reply:
I agree we should do it the way that they provide the final files.

Why don't we also add a workflow that actually does this reference building step. So that ppl can build it. Should also take care of issue #65

Reply:
@williamrowell I pulled this out to a new reference.wdl. I also asked them to contact you for additional resources to download. You can add more to address #65 to provide a path. I suggest we just make a public azure blob storage available that can download these resources from.